.ci/script.sh

```diff
@@ -7,7 +7,7 @@ BOOT_JDK10="jdk-10-ea+${BOOT_JDK10_VER}"
 echo "which java: $(which java)"
 ulimit -c unlimited -S
 
-gradle build -x :web:test --no-daemon --stacktrace --info
+./gradlew build -x :web:test --no-daemon --stacktrace --info
```

Comment:
Upstream repo does not allow files with execute bit set, so will likely need to change this to something like:

sh ./gradlew ...